_data/providers.json

```
@@ -98,6 +98,11 @@
       "url": "https://virtualgladiators.com/",
       "description": "Find the plugin in the control panel under the \"VG Recommended\" category and restart your server. IP and port are the same as Java."
     },
+    {
+      "name": "WiseHosting",
+      "url": "https://wisehosting.com",
+      "description": "Both auto and manual installation available. Find the GeyserMC presets from `Properties` tab in the Game Panel. Just click Install and Restart your server!  See [Crossplay FAQ](https://wisehosting.com/knowledgebase/how-to-install-geysermc-java-bedrock-crossplay-to-your-server) for additional details."
```

```suggestion
      "description": "Both auto and manual installation available. Find the GeyserMC presets from the `Properties` tab in the Game Panel. There, click `Install` and restart your server. See [Crossplay FAQ](https://wisehosting.com/knowledgebase/how-to-install-geysermc-java-bedrock-crossplay-to-your-server) for additional details."
```
Some minor changes, looks good otherwise